package.json

```diff
@@ -15,7 +15,7 @@
     "start:blockchain:client": "bash ./scripts/start-blockchain-client.sh",
     "stop:blockchain:client": "bash ./scripts/stop-blockchain-client.sh",
     "test:contracts": "npm run start:blockchain:client & truffle migrate --reset --compile-all && truffle test --network development",
-    "test:contracts:upgrade": "npm run start:blockchain:client parity & npm run generate:test:contracts && truffle migrate --reset --compile-all && truffle test ./upgrade-test/* --network integration",
+    "test:contracts:upgrade": "npm run start:blockchain:client & npm run generate:test:contracts && truffle migrate --reset --compile-all && truffle test ./upgrade-test/* --network integration",
```

Comment:
Do the upgrade tests no longer work on Parity? I can't spot a time advancement anywhere - is there another reason?

Reply:
It turns out you have to explicitly enable the Hard fork EIPs for Parity to work with them
https://github.com/paritytech/parity/commit/07c63c14f37a5be4f214dfe6acff033f3ec6f1c0